Cabal/Distribution/Simple/Setup.hs

```diff
@@ -950,9 +950,21 @@ copyCommand = CommandUI
 
       ,option "" ["destdir"]
          "directory to copy files to, prepended to installation directories"
-         copyDest (\v flags -> flags { copyDest = v })
+         copyDest (\v flags -> case copyDest flags of
+                      NoFlag -> flags { copyDest = v }
+                      Flag NoCopyDest -> flags { copyDest = v }
+                      _ -> error "Use either 'destdir' or 'target-package-db'.")
```

Comment:
> Use either 'destdir' or 'target-package-db'

, but not both? I think that this breaks `--destdir=foo --destdir=bar`, which used to work.

Reply:
Well, if `--target-package-db` is given, `--destdir` makes no sense. But allowing to specify `--destdir` twice (that would just take the latter one, I believe), if that used to be the behaviour should definitely stay that way.